src/components/past-events/past-events.jsx

```diff
@@ -0,0 +1,211 @@
+import * as React from 'react';
+import styled, { withTheme } from 'styled-components';
+import { media } from '../../utils/css-utils';
+
+const Header = styled.h2`
+    font-family: Rubik;
+    font-size: 48px;
+    font-weight: 500;
+    color: ${(props) => props.theme.lipstick};
+    text-align: center;
+    vertical-align: middle;
+    line-height: 100px;
+    ${media.tablet`line-height: 200px;`}
+    ${media.hd`line-height: 300px;`}
+    margin-bottom: 50px;
+    
+    &:before, &:after {
```

Comment:
This is a separator, maybe it would be better to use Nikita's component when he push it